src/components/Layout.tsx

```diff
@@ -12,7 +12,7 @@ export default function Layout(props: ParentProps) {
         <div class="grid lg:grid-cols-[1fr1fr] grid-cols-2 py-2 px-8 items-center w-full max-w-8xl mx-auto ">
           <div class="flex justify-start gap-2">
             <div class="flex justify-start gap-2">
-              <A href="/" aria-label="Home page">
+              <A href={user() ? "/" : "#"} aria-label="Home page">
```

Comment:
I'm not sure we would need this since we could rely on the redirect if no user? I guess either way works

Reply:
yeah, i kinda regret this already. I'm taking it out

Reply:
If we had more logged out pages this wouldn't quite work as intended